src/chrome/content/rules/Deutsche-Rentenversicherung.de.xml

```diff
@@ -0,0 +1,11 @@
+<ruleset name="Deutsche-Rentenversicherung.de">
+	<target host="www.deutsche-rentenversicherung.de" />
+	<target host=    "deutsche-rentenversicherung.de" />
+	<target host="www.deutsche-rentenversicherung-bund.de" />
+	<target host=    "deutsche-rentenversicherung-bund.de" />
+	<target host="www.drv-bund.de" />
+	<target host=    "drv-bund.de" />
+
+	<rule from="^https?://(?:www\.)?(?:deutsche-rentenversicherung\.de|deutsche-rentenversicherung-bund\.de|drv-bund\.de)/"
```

Comment:
Maybe so, in which case (-bund)? is probably premature optimisation. But removing the triple repeated \.de seems sensible at least. It's still readable.

Computer optimisation requires CPU time too